### PR TITLE
Deathlink button!

### PR DESCRIPTION
**Pull request by silasary**
It works!

Probably needs some kind of audio alert for when you recieve one, because button going red isn't overly attention grabbing.  But it all works.

Tested in a 2-player game with a Factorio, successfully sent and received deaths

### CODE REVIEW

File: schemas/Manual.game.schema.json [hunk 1]
```diff
@@ -28,6 +28,10 @@
                 "$ref": "#/definitions/Sitems"
             }
         },
+        "death_link": {
+            "description": "(Optional) Does your game have support Deathlink?",
```
Comment (nicopop) [inline schemas/Manual.game.schema.json:32]: might want to reRead that line ;)

File: src/data/game.json [hunk 2]
```diff
@@ -22,5 +23,6 @@
         {
             "random": 3
         }
-    ]
-}
\ No newline at end of file
+    ],
+    "death_link": true
```
Comment (nicopop) [inline src/data/game.json:27]: lets maybe not have it enabled by default? I feel like it should be something the creator has to choose to turn on

**Comment by silasary** [comment]
My reasoning is mostly that Deathlink doesn't work for every kind of game, and presenting the yaml option for something like Solitaire would be confusing.  

I'm happy with it being default-on if you'd prefer (it was on-by-default before nico's feedback), but I think giving the world creator the choice of whether Deathlink makes sense for their world is a good idea.

**Comment by FuzzyGamesOn** [comment]
> My reasoning is mostly that Deathlink doesn't work for every kind of game, and presenting the yaml option for something like Solitaire would be confusing to the end user.
> 
> I'm happy with it being default-on if you'd prefer (it was on-by-default before nico's feedback), but I think giving the world creator the choice of whether Deathlink makes sense for their world is a good idea.

That makes sense, yeah. Hard to know if people would want this available by default or gatekept by the world creator / anyone modifying the world. Without knowing, opt-in seems fine. We can always change it later based on feedback. 😃

**Review by FuzzyGamesOn — APPROVED** [review]

**Pull request merged**